channel.go

```diff
@@ -13,39 +13,54 @@ import (
 	"perun.network/go-perun/client"
 )
 
+// State wraps a go-perun/channel.State
+// ref https://pkg.go.dev/perun.network/go-perun@v0.2.2/channel?tab=doc#State
 type State struct {
 	s *channel.State
 }
 
+// GetID returns the immutable id of the channel this state belongs to.
 func (s *State) GetID() []byte {
 	return s.s.ID[:]
 }
 
+// GetVersion returns the version counter.
 func (s *State) GetVersion() int64 {
 	return int64(s.s.Version)
 }
 
+// GetBalances returns a BigInts with length two containing the current
+// asset allocation.
```

Comment:
```suggestion
// balances.
```
I think we don't use the word "allocation" anywhere else in prnm, but balances.

Reply:
I basically copied the doc from the `State.Balance` field where it says 'allocation of assets'.

Reply:
Which makes sense in the general context where the `Balance` field is a `[][]*big.Int`, so it describes the allocation of different assets and `Allocation` also has an `Assets` field. In our case, however, there's only one asset and the asset is invisible to the user. All the user sees are two numbers. So imo "balances" is less confusing in this simplified context.